src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0000</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Ignition Off
+                        <br></br>
+                        1 = Ignition On
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Information Type 1 - Modem Availability</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0000</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Modem is not ready
+                        <br></br>
+                        1 = Modem is available
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>GO Multi-Frame Data (0x27)</h3>
+        <p>Sent from the GO device to the IOX when the GO device wants to transfer data that does not fit into a single CAN frame. The first frame contains the Type and Length. All frames start with a Frame Counter that is an incrementing sequence number. The first frame always starts with 0x00.</p>
+        <h4>Payload First Frame</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter (0x00)</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Info Type</td>
+                </tr>
+                <tr>
+                    <td>2-3</td>
+                    <td>Data Length</td>
+                </tr>
+                <tr>
+                    <td>4-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Payload Subsequent Frames</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter</td>
+                </tr>
+                <tr>
+                    <td>1-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Info Types</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on